examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1233,11 +1270,11 @@ def load_model_hook(models, input_dir):
     # pooled text embeddings
     # time ids
 
-    def compute_time_ids():
+    def compute_time_ids(crops_coords_top_left, original_size=None):
         # Adapted from pipeline.StableDiffusionXLPipeline._get_add_time_ids
-        original_size = (args.resolution, args.resolution)
+        if original_size is None:
```

Comment:
Could you show me a situation where this case might arise?

Reply:
At first I thought we'd still need to call `compute_time_ids` from outside the training loop but we won't. 
removed the if condition now, thanks!